docs/conf.py

```diff
@@ -30,7 +31,9 @@
 source_suffix = ".rst"
 master_doc = "index"
 project = project_data.get("name").upper()
-year = datetime.datetime.now().date().year
+year = datetime.datetime.utcfromtimestamp(
+    int(os.environ.get("SOURCE_DATE_EPOCH", time.time()))
+).year
```

Comment:
This change yields the following deprecation warning:

> DeprecationWarning: `datetime.datetime.utcfromtimestamp()` is deprecated and scheduled for removal in a future version. Use timezone-aware objects to represent datetimes in UTC: `datetime.datetime.fromtimestamp(timestamp, datetime.UTC)`

Perhaps the following instead?

```python
year = datetime.datetime.fromtimestamp(
    int(os.environ.get("SOURCE_DATE_EPOCH", time.time())), datetime.timezone.utc
).year
```

See also:
- https://docs.python.org/3/library/datetime.html#datetime.datetime.utcfromtimestamp
- https://til.simonwillison.net/python/utc-warning-fix